site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -40,6 +56,11 @@ For example:
 ```bash
 ./sdk-server.linux.amd64 --local
 ```
+or
+```bash
+go run cmd/sdk-server/main.go --local
+```
```

Comment:
```suggestion
```

Since we have the section above now.

Reply:
Cleaned up any `go run` outside of the "run from source" section.